tools/serve/serve.py

```diff
@@ -313,16 +369,36 @@ class AnyWorkerHandler(WrapperHandler):
     def _meta_replacement(self, key, value):
         return None
 
+    def _create_script_import(self, attribute):
+        return 'importScripts("%s")' % attribute
+
     def _script_replacement(self, key, value):
         if key == "script":
             attribute = value.replace("\\", "\\\\").replace('"', '\\"')
-            return 'importScripts("%s")' % attribute
+            return self._create_script_import(attribute)
         if key == "title":
             value = value.replace("\\", "\\\\").replace('"', '\\"')
             return 'self.META_TITLE = "%s";' % value
         return None
 
 
+class AnyWorkerModuleHandler(AnyWorkerHandler):
```

Comment:
I feel like I'd prefer if this and `AnyWorkerHandler` both inherited from a common base rather than having this one override parts of the behaviour in the other concrete class.